Assets/Dojo/Runtime/Starknet/Account.cs

```diff
@@ -94,9 +94,16 @@ private unsafe Account(dojo.Account* account, SigningKey signingKey)
         // webgl js interop starknet bindings
         public async Task<FieldElement> ExecuteRaw(dojo.Call[] calls)
         {
-            var res = await StarknetInterop.AccountExecuteRawAsync(await account.Task, calls);
-
-            return res;
+            try
+                {
+                    var res = await StarknetInterop.AccountExecuteRawAsync(await account.Task, calls);
+                    return res;
+                 }
+                 catch (Exception e)
+                {
+                    Debug.LogError($"Error in ExecuteRaw: {e.Message}");
```

Comment:
I feel like if we end up wrapping the error by a try on the JS side and returning a Result then we shouldn't even throw. Since we wouldn't necessarily want to stop the game loop just because a transaction failed. We could maybe make ExecuteRaw return if the transaction failed?

Reply:
Oh yes, you got a point.

I could wrap the result of the execution on this class:

``` c#
public class Result<T>
{
        public bool IsSuccess { get; }
        public T Value { get; }
        public string Error { get; }

        private Result(T value, string error, bool isSuccess)
        {
            Value = value;
            Error = error;
            IsSuccess = isSuccess;
        }

        public static Result<T> Success(T value) => new Result<T>(value, null, true);
        public static Result<T> Failure(string error) => new Result<T>(default(T), error, false);
 }
   ```
    
What do you think?